input/fsh/rulesets/DEF_Rulesets.fsh

```diff
@@ -125,3 +125,30 @@ RuleSet: CategorySlicingRules
 * category ^slicing.rules = #open
 * category ^slicing.description = "Slicing requires the given value but allows additional categories"
 * category contains requiredCategory 1..1
+
+RuleSet: childName
+* name MS
+  * family 1.. MS
+    * extension[dataAbsentReason] ^short = "When \"name not chosen\" use code \"temp-unknown\""
+  * given MS
+    * extension[dataAbsentReason] ^short = "When \"name not chosen\" use code \"temp-unknown\""
+
+RuleSet: birthDateAndTime 
+* birthDate 1..
+  * extension[birthTime] MS 
+  * extension[partialDate] MS 
+  // * extension[datePartAbsentReason] MS 
+
+RuleSet: multipleBirths
+* multipleBirth[x] MS
+  * ^short = "If not single birth - born first, second, third, etc."
+  * ^definition = "If not single birth, specify born 1st, 2nd, etc. – For multiple deliveries, the order this infant was delivered in the set. Include all live births and fetal losses."
+// Add Edit Flags, a la VRDR 
+* multipleBirth[x].extension contains
+    BypassEditFlag named bypassEditFlag 0..1
+* multipleBirth[x].extension[bypassEditFlag]
+  * value[x] from ValueSetPluralityEditFlagsVitalRecords (required)
+  * value[x] only CodeableConcept
+  * value[x]
+    * ^short = "To reflect the relevant edit possibilities for plurality."
+    * ^binding.description = "Plurality Edit Flags (NCHS)"
```

Comment:
I don't think this is right...
MultipleBirth should have two expensions:
  1) edit flag
  2) [http://hl7.org/fhir/StructureDefinition/patient-multipleBirthTotal](http://hl7.org/fhir/extensions/1.0.0/StructureDefinition-patient-multipleBirthTotal.html)